### PR TITLE
v8.0.0-beta.3

### PR DESCRIPTION
**Pull request by skjnldsv**
- [x] Waiting for changelog

### CODE REVIEW

**Review by Pytal — APPROVED** [review]

**Comment by skjnldsv** [comment]
```
> @nextcloud/vue@8.0.0-beta.3 build:module
> vite --mode production build

vite v4.4.9 building for production...
"build.lib.formats" will be ignored because "build.rollupOptions.output" is already an array format.
transforming...
✓ 121 modules transformed.
Expected number in radix 2
file: /home/runner/work/nextcloud-vue/nextcloud-vue/src/utils/ScopeComponent.js:33:47
31:   // Inject root scoping function
32:   Component.mounted.push(function() {
33:     this.$el.setAttribute(`data-v-${SCOPE_VERSION}`, '')
                                                     ^
34:   })
35: }
error during build:
RollupError: Expected number in radix 2
    at error (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:2245:30)
    at Module.error (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:13604:16)
    at Module.tryParse (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:14330:25)
    at Module.setSource (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:13931:39)
    at ModuleLoader.addModuleSource (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:24463:20)
✓ built in 3.38s
Error: Process completed with exit code [1](https://github.com/nextcloud-libraries/nextcloud-vue/actions/runs/5882058085/job/15951784704?pr=4428#step:8:1).
```

@susnux

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 All notable changes to this project will be documented in this file.
 
 ## v8 **Unreleased** (beta)
-[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-vue/compare/v7.11.2...v8.0.0-beta.2)
+[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-vue/compare/v7.12.2...v8.0.0-beta.3)
```
Comment (susnux) [inline CHANGELOG.md:6]:
```suggestion
[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-vue/compare/v7.11.2...v8.0.0-beta.3)
```

Base version is still 7.11.2

**Comment by susnux** [comment]
Automation...

File: CHANGELOG.md [hunk 2]
```diff
@@ -60,6 +62,16 @@ All notable changes to this project will be documented in this file.
...
+- fix\(NcModal\): Close button should be visible even if modal content is scrolled [\#4350](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4350) ([susnux](https://github.com/susnux))
+- fix\(NcButton\): Improve alignment documentation to show all variants. [\#4392](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4392) ([susnux](https://github.com/susnux))
+- try to fix header size on mobile [\#4420](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4420) ([szaimen](https://github.com/szaimen))
+- fix\(NcButton\): correctly emit click event [\#4398](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4398) ([raimund-schluessler](https://github.com/raimund-schluessler))
+- feat\(NcButton\): Add `pressed` state for stateful buttons [\#4344](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4344) ([susnux](https://github.com/susnux))
+- NcCheckboxRadioSwitch - adjust border color [\#4427](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4427) ([szaimen](https://github.com/szaimen))
```
Comment (susnux) [inline CHANGELOG.md:75]:
```suggestion
- fix\(NcRichContenteditable\): Add aria-placeholder [\#4407](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4407) ([ShGKme](https://github.com/ShGKme))
- fix\(NcModal\): Fix modal focus trap with additional elements and arrow navigation [\#4406](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4406) ([ShGKme](https://github.com/ShGKme))

```

Comment (susnux) [reply CHANGELOG.md:75]: Because of the rebase for the build fix.

**Comment by raimund-schluessler** [comment]
> ```
> > @nextcloud/vue@8.0.0-beta.3 build:module
> > vite --mode production build
> 
> vite v4.4.9 building for production...
> "build.lib.formats" will be ignored because "build.rollupOptions.output" is already an array format.
> transforming...
> ✓ 121 modules transformed.
> Expected number in radix 2
> file: /home/runner/work/nextcloud-vue/nextcloud-vue/src/utils/ScopeComponent.js:33:47
> 31:   // Inject root scoping function
> 32:   Component.mounted.push(function() {
> 33:     this.$el.setAttribute(`data-v-${SCOPE_VERSION}`, '')
>                                                      ^
> 34:   })
> 35: }
> error during build:
> RollupError: Expected number in radix 2
>     at error (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:2245:30)
>     at Module.error (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:13604:16)
>     at Module.tryParse (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:14330:25)
>     at Module.setSource (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:13931:39)
>     at ModuleLoader.addModuleSource (file:///home/runner/work/nextcloud-vue/nextcloud-vue/node_modules/rollup/dist/es/shared/node-entry.js:24463:20)
> ✓ built in 3.38s
> Error: Process completed with exit code [1](https://github.com/nextcloud-libraries/nextcloud-vue/actions/runs/5882058085/job/15951784704?pr=4428#step:8:1).
> ```
> 
> @susnux

I also see this. Weirdly, this line compiles to
```
this.$el.setAttribute(`data-v-${f7c85e6}`, "");
```
with `f7c85e6` as a variable which is obviously not defined.

**Pull request merged**